api/numaresourcesoperator/v1/helper/nodegroup/nodegroup.go

```diff
@@ -26,6 +26,7 @@ import (
 	mcov1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
 
 	nropv1 "github.com/openshift-kni/numaresources-operator/api/numaresourcesoperator/v1"
+	"github.com/openshift-kni/numaresources-operator/internal/api/annotations"
```

Comment:
we can't import `internal` packages in public packages. We will need to review the code arrangement.

Reply:
good idea to do it in the controller, turned out to be much fewer changes; Thanks for highlighting this!